### ISSUE TITLE
Why is model not set to `eval` mode during inference

### ISSUE THREAD

**Issue by chenzhekl** [SimonGiebenhain/pixel3dmm#19]
Hi,

Thanks for open-sourcing the great work.

I read through the code and found that the model was set to `train` mode during inference.

https://github.com/SimonGiebenhain/pixel3dmm/blob/fcd1fa973c7715b02a8948dfc679dff53cf85924/scripts/network_inference.py#L109-L110

I was wondering if it's an intended design choice.

**Comment by chenzhekl** [comment]
There seems to be no randomness in the model (no batchnorm, no activated dropout). So leaving the model in `train` mode might be fine. Sorry for the noise.
